spec/models/product_file_spec.rb

```diff
@@ -815,6 +815,53 @@
     end
   end
 
+  describe "isbn" do
```

Comment:
Can you also test for an ISBN-10 value? I only see ISBN-13 formats being tested here.